crates/networking/discv5/src/discovery.rs

```diff
@@ -66,9 +66,31 @@ pub struct Discovery {
     discovery_queries: FuturesUnordered<Pin<Box<dyn Future<Output = QueryResult> + Send>>>,
     find_peer_active: bool,
     pub started: bool,
+    attestation_subnets: RwLock<AttestationSubnets>,
+    sync_committee_subnets: RwLock<SyncCommitteeSubnets>,
```

Comment:
please remove these, and instead update the ones in 
<img width="463" alt="image" src="https://github.com/user-attachments/assets/a20374a8-4ba6-4766-984e-ab4f9a5200ae" />

both the subnets are stored here, and need to be updated here